Updated Using Backstage documentation for Feature Issue 21943 Write a UI Guide (#31842)
backstage/backstage: docs/getting-started/view-what-you-own.md

---
id: view-what-you-own
title: Viewing what you own
description: View the entities that you own either directly or through a group
---

Audience: Developers

> You can own entities either directly or through a group of which you are a part.

Reviewer: ```suggestion
You can own entities either directly or through a group that you're part of.
```
A little simpler I think?
